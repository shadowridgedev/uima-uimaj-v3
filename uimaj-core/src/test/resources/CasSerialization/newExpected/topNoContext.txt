{"@cas_feature_structures":{"1":{"@type":"TOP"}},"@cas_views":{"0":[1]}}