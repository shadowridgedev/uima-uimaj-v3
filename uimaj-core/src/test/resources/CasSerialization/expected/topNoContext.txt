{"_cas_feature_structures":{"1":{"TOP":{}}},"_cas_views":{"0":[1]}}